edge-modules/load-gen/Settings.cs

```diff
@@ -74,6 +77,8 @@ public class Settings
 
         public string TrackingId { get; }
 
+        public Uri TestResultCoordinatorUrl { get; }
```

Comment:
Need to avoid using null.
```suggestion
        public Option<Uri> TestResultCoordinatorUrl { get; }
```